src/__tests__/mock-ig/example-elm.json

```diff
@@ -0,0 +1,62 @@
+{
+  "library": {
+    "annotation": [
+      { "translatorOptions": "", "type": "CqlToElmInfo" },
+      {
+        "libraryId": "example",
+        "libraryVersion": "0.0.1",
+        "startLine": 12,
+        "startChar": 2,
+        "endLine": 12,
+        "endChar": 42,
+        "message": "Could not resolve membership operator for terminology target of the retrieve.",
```

Comment:
Is this defining a mock for an expected outcome of a valid or invalid response.  The annotation, while a warning, seems like it could be an issue with the CQL.  Is this an issue due to the version of the translation service or something else?

Reply:
This is the result I got back from running some mock CQL through the cql translation service. The response is "valid" per se since there are no messages w "error" as the severity.

FWIW, I see messages like this in basically all the CQL I've translated with it